### PR TITLE
@Ignore Intermittent clustering test failures

### PR DESCRIPTION
**Pull request by ctomc**
@pferraro please take a look at this.

stats:
http://brontes.lab.eng.brq.redhat.com/project.html?projectId=WF_PullRequest&testNameId=1140899232324454378&tab=testDetails
http://brontes.lab.eng.brq.redhat.com/project.html?projectId=WF&testNameId=-2142179837056604036&tab=testDetails

one jira already exists for this https://issues.jboss.org/browse/WFLY-5239

### CODE REVIEW

File: testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/xsite/XSiteSimpleTestCase.java [hunk 1]
```diff
@@ -114,6 +115,7 @@
      *   arrives at SFO-0 on site SFO
      */
     @Test
+    @Ignore
```
Comment (bstansberry) [inline testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/xsite/XSiteSimpleTestCase.java]: Not in the commit message, as the commit does not fix the bug.

Comment (rhusar) [inline testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/xsite/XSiteSimpleTestCase.java]: Ok, not the commit message then :-)

**Comment by pferraro** [comment]
Huh?  I'm not seeing failures for CoarseSessionExpirationTestCase in master.

**Comment by rhusar** [comment]
I think @ctomc is accidentally looking at the wrong queue, the correct queue (master) has no failure within last month:
http://brontes.lab.eng.brq.redhat.com/project.html?tab=testDetails&testNameId=-2142179837056604036&projectId=WF&buildTypeId=WF_MasterLinux

**Comment by ctomc** [comment]
@pferraro I updated the link for CoarseSessionExpirationTestCase to include all all jobs not just PR.
But statistically speaking, PR jobs will be the ones that will get catch intermittent problems more often.
as usually we have something like 100+ PR jobs executions per 1 master job execution. 

There can be some margin of error for this as given PR itself can cause failure, but in most cases failure is unrelated to change PR is doing.

**Comment by pferraro** [comment]
But that makes the assumption that the PR is question isn't breaking the test, no?  That was indeed the case with https://github.com/wildfly/wildfly/pull/8021, until I fixed it.

**Comment by pferraro** [comment]
@ctomc I see you point, but I'm not seeing any recent intermittent failures for this test that aren't clearly an issue with the associated PR.

**Comment by bstansberry** [comment]
Closing, since the first commit looks unneeded. 

@rhusar or @pferraro please send up a PR to ignore xsite with whatever @Ignore text value you want. Per the email from the CI Inquisitor[1] it's not the job of the person who is impacted by failing tests (in this case, Tomaz) to tailor the @Ignore message. The test just gets ignored with minimum effort and the test owner deals with it.

[1] http://lists.jboss.org/pipermail/wildfly-dev/2015-July/004246.html

**Comment by rhusar** [comment]
Opened https://github.com/wildfly/wildfly/pull/8040 to ignore the latter.

**Pull request closed without merging**